plugins/karate/Makefile.mk

```diff
@@ -1,8 +1,14 @@
 # LIBRARIES
 ##################################################
 if USE_KARATE
+# Plugin description is generated from README.md
+BUILT_SOURCES += plugins/karate/PluginDescription.h
```

Comment:
Can you switch the name to KaratePluginDescription please (which I can ask for now it's in each file)